spec/presenters/organisations/documents_presenter_spec.rb

```diff
@@ -49,6 +51,47 @@
       expect(documents_presenter.remaining_featured_news).to eq(expected)
     end
 
+    it "formats news stories correctly if No. 10" do
```

Comment:
Small suggestion but not a blocker. I'd wrap this section in a `context "for No 10" do` and move the two `let` additions into that block.